module.yml

```diff
@@ -354,6 +401,73 @@ Resources:
           Condition:
             StringEquals:
               's3:x-amz-acl': 'bucket-owner-full-control'
+  BucketPolicyCloudTrailCrossAccountWrite:
```

Comment:
Is there a way to restrict which AWS account is able to write logs? For example, by specifying the account or OU id?

Reply:
removed